app/controllers/api/v1/code_schools_controller.rb

```diff
@@ -2,7 +2,50 @@ module Api
   module V1
     class CodeSchoolsController < ApplicationController
       def index
-        render json: CodeSchools.all
+        render json: CodeSchool.all
+      end
+      
+      def create 
+        school = CodeSchool.new(code_school_params)
+        
+        if school.save 
+          render json: { code_school: school.id }
+        else 
+          render json: { :errors => school.errors.full_messages }
```

Comment:
Can you change these to the newer `errors: school.errors.full_messages` syntax.

Reply:
@hpjaj yeah, sorry about that